iwyu.cc

```diff
@@ -2604,6 +2602,17 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     return retval;
   }
 
+ private:
```

Comment:
I don't think it did move.  The diff is just confused because I inserted new code above it in the protected: section.